### DISCUSSION TITLE
Where does the name on the module list come from and why does it have my name?

### DISCUSSION THREAD

**Discussion by zhengyi0533** [LSPosed/LSPosed#802] [Q&A] [answered]
![IMG_20210703_161827](https://user-images.githubusercontent.com/71653166/124348971-75992600-dc1f-11eb-9c7c-82eb671d436f.jpg)
Where does the name on the module list come from and why does it have my name? Thanks! I can't find it from my phone.

**Comment by vvb2060** [comment, marked as answer]
https://github.com/LSPosed/LSPosed/releases/tag/v1.4.3

**Reply by zhengyi0533** [reply]
The 0. XML file  is generated by lsposed！I changed the content inside, and after I reloaded lsposed, the content was restored. Can you tell me where lsposed reads the user name file？Thanks！

**Reply by yujincheng08** [reply]
We are open-sourced. You can check that this file is not generated by LSPosed, but by your system.

**Reply by zhengyi0533** [reply]
Thank you！

**Reply by zhengyi0533** [reply]
> We are open-sourced. You can check that this file is not generated by LSPosed, but by your system.

It's really system generated

**Reply by shq001** [reply]
Have you found the specific reason?请问你找到具体原因了吗

**Comment by zhengyi0533** [comment]
![image](https://user-images.githubusercontent.com/71653166/124350746-09232480-dc29-11eb-9664-26048cef3c23.png)
My phone doesn't have multi-user settings. I just found my name by browsing /data/system/users/0.xml. Is the 0. XML file generated by lsposed? Can you tell me where this information comes from? I remember I didn't put my name in my phone. Thank you very much！

**Comment by canyie** [comment]
Disable "MIUI Optimization" from Developer Options

**Reply by zhengyi0533** [reply]
Thank you! But useless

**Reply by zhengyi0533** [reply]
![image](https://user-images.githubusercontent.com/71653166/124360118-8915b280-dc5a-11eb-8866-65a2fae4f2f9.png)
After being reminded, I found multiple users. It's in the first level menu, and I've been looking for more settings in the second level menu.Thank you for your help！

**Reply by shq001** [reply]
> 从开发者选项禁用“MIUI 优化”

Does this sentence mean that Xiaomi stole my privacy?这句话的意思是小米窃取了我的隐私吗

**Comment by yujincheng08** [comment]
https://github.com/LSPosed/LSPosed/blob/8bb287ae08b12272e9f3a255993a6326ee93c5d4/app/src/main/java/org/lsposed/manager/ui/fragment/ModulesFragment.java#L192

We get user list from this API: https://cs.android.com/android/platform/superproject/+/master:frameworks/base/core/java/android/os/UserManager.java;l=3257
